after/plugin/dracula.vim

```diff
@@ -41,5 +41,66 @@ if exists('g:loaded_ctrlp')
   hi! link CtrlPBufferHid Normal
 endif
 " }}}
+" Tree-sitter: {{{
+if exists('g:loaded_nvim_treesitter')
+  " # Misc
+  hi! link TSError DraculaRed
+  hi! link TSPunctDelimiter Normal
+  hi! link TSPunctBracket Normal
+  hi! link TSPunctSpecial Normal
+  " # Constants
+  hi! link TSConstant DraculaPurple
+  hi! link TSConstBuiltin DraculaPurple
+  hi! link TSConstMacro DraculaCyan
+  hi! link TSStringRegex DraculaYellow
+  hi! link TSString DraculaYellow
+  hi! link TSStringEscape DraculaPink
+  hi! link TSSymbol DraculaPurple
+  hi! link TSCharacter DraculaGreen
+  hi! link TSNumber DraculaPurple
+  hi! link TSBoolean DraculaPurple
+  hi! link TSFloat DraculaPurple
+  hi! link TSAnnotation DraculaYellow
+  hi! link TSAttribute DraculaGreenItalic
+  hi! link TSNamespace DraculaPink
+  " # Functions
+  hi! link TSFuncBuiltin DraculaCyan
+  hi! link TSFunction DraculaGreen
+  hi! link TSFuncMacro DraculaGreen
+  hi! link TSParameter DraculaOrangeItalic
+  hi! link TSParameterReference DraculaOrange
+  hi! link TSMethod DraculaGreen
+  hi! link TSField DraculaOrange
+  hi! link TSProperty Normal
+  hi! link TSConstructor DraculaCyan
+  " # Keywords
+  hi! link TSConditional DraculaPink
+  hi! link TSRepeat DraculaPink
+  hi! link TSLabel DraculaPurpleItalic
+  hi! link TSKeyword DraculaPink
+  hi! link TSKeywordFunction DraculaCyan
+  hi! link TSKeywordOperator DraculaPink
+  hi! link TSOperator DraculaPink
+  hi! link TSException DraculaPurple
+  hi! link TSType DraculaCyanItalic
+  hi! link TSTypeBuiltin DraculaCyanItalic
+  hi! link TSStructure DraculaPurple
+  hi! link TSInclude DraculaPink
+  " # Variable
+  hi! link TSVariable Normal
+  hi! link TSVariableBuiltin DraculaPurpleItalic
+  " # Text
+  hi! link TSText DraculaFg
+  hi! link TSStrong DraculaFg
+  hi! link TSEmphasis DraculaFg
+  hi! link TSUnderline DraculaFg
+  hi! link TSTitle DraculaYellow
+  hi! link TSLiteral DraculaYellow
```

Comment:
Not sure what these are specifically, but headings are purple and bold and inline code delimited with backticks is green.